VTUL/dlp-access
# Add file characterization for podcast audio files

**Add file characterization for podcast audio files.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2102) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Add file characterization for podcast audio files.

# What's the changes? (:star:)
* Gets name, type, and size (bytes) from File object
* Creates Audio object from file to get duration (time)

# How should this be tested?
* Go to "/podcastDeposit" page
* Fill in values for all fields
* Upload an audio file (there is a test file you can use at `cypress/fixtures/sitecontent/audioFile.wav`)
* Submit the form
* On the page you're taken to you should see "Podcast entry successfully created"
* Check to make sure that the file details show up under "manifest_file_characterization:"

# Additional Notes:
* branch: `LIBTD-2102`

# Interested parties
@yinlinchen 

(:star:) Required fields
